src/networkUtils.ts

```diff
@@ -447,9 +439,48 @@ function getDepositAddress(from: Network|string, to: Network|string, destination
     return address;
 }
 
+const chains = ["moonbeam", "avalanche", "fantom", "ethereum", "polygon"];
+const createLocal = async (userPrivateKey: string, deployerPrivateKey: string, options: CreateLocalOptions = {
```

Comment:
this should be named something that describes what it does better, maybe "createAndExport"

Reply:
userPrivateKey should be userAddress here, so that users can pass their metamask potentially.